symd.py

```diff
@@ -330,7 +336,10 @@ def print_dependency_tree_as_json(graph=None, filename=None, yang_dict={}):
     output['nodes'] = []
     output['links'] = []
     idx_arr = []
-    ignore_exact = [ 'ietf-yang-types', 'ietf-inet-types', 'toaster', 'toaster-provider']
+    """
+    Fix the bug to avoid the "None" object.  We modify the ignore_exact with "None".
```

Comment:
Rather than having a comment "Fix the bug...", can we have a comment that describes why the `None` entry is needed on the list, as that may be counter-intuitive at face value.